src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -381,11 +382,17 @@ public void testSnapshotBuilder() throws Exception {
     var getConceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
     List<String> getConceptNames =
         getConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
-    assertThat("Correct number of concepts are returned", getConceptNames, hasSize(2));
     assertThat(
         "expected concepts are returned",
         getConceptNames,
         containsInAnyOrder("concept1", "concept3"));
+
+    // Test searchConcepts
+    var searchConceptResponse =
+        dataRepoFixtures.searchConcepts(steward, datasetId, "Condition", "concept1");
+    List<String> searchConceptNames =
+        searchConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
+    assertThat("expected concepts are returned", searchConceptNames, contains("concept1"));
```

Comment:
Given that it's a search, should we also check that `concept3` is not returned?

Reply:
from the contains documentation: Creates a matcher for Iterables that matches when a single pass over the examined Iterable yields a series of items, each logically equal to the corresponding item in the specified items. For a positive match, the examined iterable must be of the same length as the number of specified items. For example:
assertThat(Arrays.asList("foo", "bar"), contains("foo", "bar")).

Since it verifies length I think that is covered.